scripts/async_profiler_build_shared.sh

```diff
@@ -5,12 +5,27 @@
 #
 set -euo pipefail
 
-VERSION=v2.8.3g2
-GIT_REV="ef1d3b3c29e7337d52f54035a12ae9a90eb1c710"
+VERSION=v2.8.3g3
+GIT_REV="008a8cf435bedce8eec3ef36fdc67e742b070566"
 
 git clone --depth 1 -b "$VERSION" https://github.com/Granulate/async-profiler.git && cd async-profiler && git reset --hard "$GIT_REV"
 source "$1"
 make all
 
 # add a version file to the build directory
 echo -n "$VERSION" > build/async-profiler-version
+
+# build verifications regarding libstdc++.
+if ldd /bin/ls 2>&1 | grep -q musl ; then
```

Comment:
Why do we need those build time tests? The CI runs on centos 6 and thus verifies that we don't use new symbols

And we have a test in `test_java.py` that makes sure that on musl we don't load libstd++

Reply:
(CentOS 6 test only tests gProfiler, not async-profiler loading on CentOS 6)